pkg/jsonnet/implementation/implementation.go

```diff
@@ -12,5 +14,11 @@ func Get(name string) (types.JsonnetImplementation, error) {
 		return &goimpl.JsonnetGoImplementation{}, nil
 	}
 
+	if strings.HasPrefix(name, "binary:") {
+		return &binary.JsonnetBinaryImplementation{
+			BinPath: strings.TrimPrefix(name, "binary:"),
+		}, nil
```

Comment:
Check the thing exists and is executable here, return a nice error message if it's not

Reply:
Done!